data-style-guide.md

```diff
@@ -104,71 +62,110 @@ The vertical axis should start from zero where the metric is a percentage.
 
 When there are few lines to compare but they are very close together, they can be presented in a line chart (rather than multiple line charts) but changing the minimum value in the y axis. However, testing has shown this approach is less easily read. Read how this is done in the section ‘Chart values and labels’ in this guide.
 
-## Ethnic categories
+## Data
 
-Ethnic categories should follow those listed in the [style guide A to Z](/a-z) where possible.
+### Special symbols
 
-If departments have submitted different categories, amend them to match the [style guide A to Z](/a-z) if you can. For example, if the original dataset uses ‘White’ and ‘BME’, rename ‘BME’ to ‘Other’.
+There should not be any empty cells in charts, tables and data downloads.
 
-If the original dataset uses categories that don’t fit with any of the groupings in the [style guide A to Z](/a-z), don’t update them. (For example, [Internet Use](https://www.ethnicity-facts-figures.service.gov.uk/culture-and-community/digital/internet-use/latest) uses the following 4 ethnic categories: Asian, Black, White, Other (including Mixed)).
+Use the following symbols if data is missing or has been rounded to zero:
 
-Make sure the ethnic categories used in a measure are the same across tables and charts. Don’t change the categories in data download files.
+<ul class="symbols-list">
+<li><kbd>!</kbd> data withheld for confidentiality</li>
+<li><kbd>?</kbd> data not shown because small sample size makes it unreliable</li>
+<li><kbd>-</kbd> data not collected [or otherwise unavailable]</li>
+<li><kbd>N/A</kbd> data not applicable (eg no meaningful numerator)</li>
+  <li><kbd>~0</kbd> the actual figure is greater than 0 but has been rounded down</li>
```

Comment:
The input symbol is just `~` which then renders as `~0`

Reply:
As discussed in Slack, we need to update the code so that '~0' triggers the footnote, rather than '~'. (Apparently analysts would naturally use '~0' when they're importing the data. 
I've created this ticket for the code update: https://trello.com/c/UHNU5asY
Thanks